tests/ui/generics/span-bug-issue-121418.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(const_trait_impl)]
```

Comment:
Could you move this test into `tests/ui/rfcs/rfc-2632-const-trait-impl/effects/`